test_type/shard.js

```diff
@@ -243,28 +241,26 @@ function printTestResult(testList) {
       totalTps += tps;
       totalTxCount += confirmJob.output.statistics.transactionCount;
       totalTimeoutTxCount += confirmJob.output.statistics.timeoutTransactionCount;
-      for (const time in confirmJob.output.statistics.confirmedTimeTable) {
-        if (confirmedTimeTable[time] === undefined) {
-          confirmedTimeTable[time] = 0;
-        }
-        confirmedTimeTable[time] += confirmJob.output.statistics.confirmedTimeTable[time];
-      }
-
       console.log(`TPS: ${Number(tps).toFixed(5)} ` +
           `(${confirmJob.output.statistics.transactionCount} txs ` +
           `/ ${confirmJob.output.statistics.blockDuration / 1000} secs)`);
-      // console.log(`Loss Rate: ${confirmJob.output.statistics.lossRate}`);
-      // console.log(`Confirmed time average (ms) : ${confirmJob.output.statistics.confirmedTimeAverage}`); // TODO(csh): Delete after test
-      // console.log(`Confirmed time table (sec) : ${JSON.stringify(confirmJob.output.statistics.confirmedTimeTable, null, 2)}`); // TODO(csh): Delete after test
     }
     console.log();
   }
-  console.log(`Total TPS : ${totalTps.toFixed(5)}`);
+  totalTps = Number(totalTps.toFixed(5));
+  const lossRate = (totalTimeoutTxCount / totalTxCount * 100).toFixed(5) + '%';
```

Comment:
How about keeping lossRate as a number like totalTps? You could add the '%' in console.log

Reply:
Looks better! I'll change it